src/App.js

```diff
@@ -1,26 +1,27 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
-import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
-  };
+export function App() {
+  const [movies, setMovies] = useState([]);
+
+  const addMovie = (movie) => {
+    if (movies.some(mov => mov.imdbID === movie.imdbID)) {
+      return;
+    }
 
-  render() {
-    const { movies } = this.state;
+    setMovies(prevMovies => [movie, ...prevMovies]);
```

Comment:
```suggestion
    setMovies([movie, ...movies]);
```
no need to use prevMovies with hooks